base_report_to_printer/views/printing_printer.xml

```diff
@@ -33,6 +33,20 @@
                     />
                 </header>
                 <sheet>
+                    <div class="oe_button_box" name="button_box">
+                        <button
+                            name="toggle_active"
+                            type="object"
+                            class="oe_stat_button"
+                            icon="fa-archive"
+                        >
+                            <field
+                                name="active"
+                                widget="boolean_button"
```

Comment:
Boolean button does not work on 13.0

Reply:
True, fixed according to the new paradigm.